x-pack/plugin/core/src/main/java/org/elasticsearch/license/internal/TrialLicenseVersion.java

```diff
@@ -27,12 +27,9 @@ public class TrialLicenseVersion implements ToXContentFragment, Writeable {
     // generic Elasticsearch version. While it's derived from the Elasticsearch version formula for BWC, it is independent of it going
     // forward. When we want users to be able to start a new trial, increment this number.
     // Pkg-private for testing only.
-    static final int TRIAL_VERSION_CUTOVER = 8_12_00_99;
+    static final int TRIAL_VERSION_CUTOVER = 9_00_00_00;
```

Comment:
I think it would be best to rename this at this point; it's not the cutover version anymore, it's the current trial version.

Also I am not asking you to change this but this version number is decoupled from the product version intentionally, so arguably the new trial version number should be `8120100`. That said, I think that's confusing in practice and matching the product version isn't a bad idea.

Reply:
Thanks! Yes, I agree. I went with keeping it as `9_00_00_00`.